From: Simon McVittie <smcv@debian.org>
Date: Sun, 10 Aug 2025 14:15:43 +0100
Subject: Revert "Replace hardcoded values of Cantarell"

In Debian, we want to keep using Cantarell until the Adwaita fonts are
packaged (<https://bugs.debian.org/1101879>).

This reverts commit 7d2d7ff553b8b8cec0e13c8e02787a9f77fbbf5b.

Forwarded: not-needed
---
 demos/gtk-demo/css_accordion.css | 2 +-
 gdk/wayland/gdkscreen-wayland.c  | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/demos/gtk-demo/css_accordion.css b/demos/gtk-demo/css_accordion.css
index 0917b44..ce3346d 100644
--- a/demos/gtk-demo/css_accordion.css
+++ b/demos/gtk-demo/css_accordion.css
@@ -4,7 +4,7 @@
     transition-property: color, background-color, border-color, background-image, padding, border-width;
     transition-duration: 1s;
 
-    font-size: 20px;
+    font: 20px Cantarell;
 }
 
 window {
diff --git a/gdk/wayland/gdkscreen-wayland.c b/gdk/wayland/gdkscreen-wayland.c
index f3450b3..71a36f4 100644
--- a/gdk/wayland/gdkscreen-wayland.c
+++ b/gdk/wayland/gdkscreen-wayland.c
@@ -528,7 +528,7 @@ static TranslationEntry translations[] = {
   { FALSE, "org.gnome.desktop.interface", "icon-theme", "gtk-icon-theme-name", G_TYPE_STRING, { .s = "gnome" } },
   { FALSE, "org.gnome.desktop.interface", "cursor-theme", "gtk-cursor-theme-name", G_TYPE_STRING, { .s = "Adwaita" } },
   { FALSE, "org.gnome.desktop.interface", "cursor-size", "gtk-cursor-theme-size", G_TYPE_INT, { .i = 32 } },
-  { FALSE, "org.gnome.desktop.interface", "font-name", "gtk-font-name", G_TYPE_STRING, { .s = "Adwaita Sans 11" } },
+  { FALSE, "org.gnome.desktop.interface", "font-name", "gtk-font-name", G_TYPE_STRING, { .s = "Cantarell 11" } },
   { FALSE, "org.gnome.desktop.interface", "cursor-blink", "gtk-cursor-blink", G_TYPE_BOOLEAN,  { .b = TRUE } },
   { FALSE, "org.gnome.desktop.interface", "cursor-blink-time", "gtk-cursor-blink-time", G_TYPE_INT, { .i = 1200 } },
   { FALSE, "org.gnome.desktop.interface", "cursor-blink-timeout", "gtk-cursor-blink-timeout", G_TYPE_INT, { .i = 3600 } },
